### PR TITLE
Frontend, Backend: Gigastake changes

### PR DESCRIPTION
**Pull request by Evalir**
Makes the Portal UI compatible with Gigastakes. Namely:
- From now on, users won't select a chain anymore when creating an app. They'll have a dropdown available on the UI to just get the endpoint for the chain they wanna use once they create it.
- Users won't see how much POKT is staked or whatnot with gigastake-based apps. They'll just get their max relays.
- Every time a new endpoint is made, a dummy Pocket account is made as well. This lets us attach metrics properly.

Screenshot:
Screen Shot 2022-01-18 at 18 09 22

### CODE REVIEW

File: packages/backend/src/controllers/LoadBalancerController.ts [hunk 1]
```diff
@@ -213,35 +227,41 @@ router.post(
...
+      publicKey: rawAccount.publicKey,
+      privateKey: encryptor.encrypt(rawAccount.privateKey),
+      passPhrase: rawAccount.passPhrase,
     }
+
+    const freeTierAAT = await PocketAAT.from(
+      '0.0.1',
+      (env('POCKET_NETWORK') as PocketNetworkKeys).clientPubKey,
+      rawAccount.publicKey,
+      rawAccount.privateKey
```
Comment (Evalir) [inline packages/backend/src/controllers/LoadBalancerController.ts:242]: One thing I didn't remember @nymd : we need to use the gateway client pub key here to make the AAT for free tier apps right?

File: packages/backend/src/known-chains.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
+export const KNOWN_CHAINS = {
```
Comment (Evalir) [inline packages/backend/src/known-chains.ts:1]: The create screen still fetches these chains to show them as "usable". So, whenever we add a new chain we can add it here too. I'd prefer if we moved this config to the DB and had a field for chains that are being serviced correctly

File: packages/backend/src/lib/pocket.ts [hunk 3]
```diff
@@ -290,6 +298,32 @@ export async function getApp(
   return applicationResponse
 }
 
+export async function createPocketAccount(): Promise<PocketAccount | Error> {
```
Comment (Evalir) [inline packages/backend/src/lib/pocket.ts:301]: One of the interesting parts of the PR. The raw pocket account's made here.

File: packages/backend/src/controllers/LoadBalancerController.ts (hunk 1 above)
Comment (nymd) [reply packages/backend/src/controllers/LoadBalancerController.ts:242]: Correct, the gateway key is always the client so the portal API can generate a session for them.

File: packages/backend/src/known-chains.ts (hunk 2 above)
Comment (nymd) [reply packages/backend/src/known-chains.ts:1]: Yes, open an issue for this, thanks.

Comment (Evalir) [reply packages/backend/src/known-chains.ts:1]: Opened an issue on the Portal API repo: https://github.com/pokt-foundation/portal-api/issues/595

**Pull request merged**